widgets/image-grid/tpl/default.php

```diff
@@ -20,9 +19,7 @@
 						<?php endif; ?>
 					<?php endforeach; ?>>
 				<?php endif; ?>
-				<?php echo wp_get_attachment_image( $image['image'], $attachment_size, false, array(
-					'title' => $image['title']
-				) );?>
+				<?php echo $image['image_html'];	?>
```

Comment:
@AlexGStapleton Can you check out the additional spaces before the PHP close when able. Thanks mate.